deps/c-ares/src/ares_parse_txt_reply.c

```diff
@@ -114,7 +114,7 @@ ares__parse_txt_reply (const unsigned char *abuf, int alen,
         }
 
       /* Check if we are really looking at a TXT record */
-      if (rr_class == C_IN && rr_type == T_TXT)
+      if ((rr_class == C_IN || rr_class == C_CHAOS) && rr_type == T_TXT)
```

Comment:
Is this change a port from upstream?

Reply:
No, that's not from upstream, so I suppose it's better if I submit it there, too.

Other than that, I'll add some tests and update this PR.

Reply:
Upstream PR got merged.